src/BenchmarkDotNet/Locators/LocatorType.cs

```diff
@@ -0,0 +1,6 @@
+namespace BenchmarkDotNet.Locators;
+
+public enum LocatorType
+{
+    ProjectFile
+}
```

Comment:
In line with the interface rename.

```suggestion
public enum FileLocatorType
{
    Project
}
```